app/models/supplejack_api/concerns/user_set.rb

```diff
@@ -95,6 +95,28 @@ def self.featured_sets(num = 16)
     def update_attributes_and_embedded(new_attributes = {}, user = nil)
       new_attributes = new_attributes.try(:symbolize_keys) || {}
 
+      update_set_items(new_attributes)
+      update_featured_set(new_attributes, user)
+
+      self.attributes = new_attributes
+
+      self.description = '' if description.nil?
```

Comment:
could do I guess.  If those methods are available in a concern?  @EdwinRozario also admitted my code is a little hacky, but agree that since the user_set endpoint will be decommissioned in the next 12 months, it doesn't matter so much.